cmd/simulator/load/loader.go

```diff
@@ -129,5 +152,51 @@ func ExecuteLoader(ctx context.Context, config config.Config) error {
 		return err
 	}
 	log.Info("Tx agents completed successfully.")
+
+	logOtherMetrics(blockchainIDStr, endpoint)
+	return nil
+}
+
+func logOtherMetrics(blockchainIDStr string, endpoint string) error {
+	getCallStart := time.Now()
+	metricsAPI := fmt.Sprintf("http://127.0.0.1:%s/ext/metrics", endpoint)
```

Comment:
we should not call the port and endpoint.

We should not add any assumptions about the format of the URIs passed in for the load test.

Reply:
What can I assume about the endpoint? I want to be able to use the `/ext/metrics` endpoint. I should be able to use that right?